.eslintrc.yml

```diff
@@ -1,54 +1,35 @@
 root: true
 parser: '@typescript-eslint/parser'
 parserOptions:
-    tsconfigRootDir: '.'
-    project: 
-        - tsconfig.json
-    ecmaVersion: 'latest'
+    project: tsconfig.json
 
-env:
-    browser: true
-    node: true
-    es6: true
-
-plugins: 
+plugins:
+    - react
+    - react-hooks
     - import
-    - simple-import-sort
```

Comment:
I kinda liked having auto import sorting. Are there any other isort plugins that we can get that don't screw up the leaflet imports? It's fine if not.

Reply:
Sure, I'll turn on this rule. https://github.com/import-js/eslint-plugin-import/blob/main/docs/rules/order.md